### ISSUE TITLE
Error on adding Zustand

### ISSUE THREAD

**Issue by aadamsx** [alephjs/aleph.js#220]
I'm trying out Zustand with Aleph but getting an error:

```
500 - Invalid hook call. Hooks can only be called inside of the body of a function component. 
This could happen for one of the following reasons: 
1. You might have mismatching versions of React and the renderer (such as React DOM) 
2. You might be breaking the Rules of Hooks 
3. You might have more than one copy of React in the same app 
4. See https://reactjs.org/link/invalid-hook-call for tips about how to debug and fix this problem.
```

```
$ aleph --version
aleph.js 0.3.0-alpha.12
deno 1.8.2
v8 9.0.257.3
typescript 4.2.2
```

My code: https://github.com/aadamsx/aleph-error

**Comment by shadowtime2000** [comment]
Seems it occurs because esm.sh uses React 17.0.1 and Zustand is using React 17.0.2

**Comment by ije** [comment]
@shadowtime2000  in fact the compiler will fix the react version of esm.sh, @aadamsx  i guess you have old cache that cause this issue, pls run it with command `aleph dev -reload`

**Comment by ije** [comment]
btw, i already released the **v0.3.0-alpha.22**, some breaking changes included...

**Comment by aadamsx** [comment]
> btw, i already released the v0.3.0-alpha.22, some breaking changes included...

Some breaking changes, changes that will effect my code you say?

**Comment by aadamsx** [comment]
> i guess you have the old cached react that does not match the Zustands, pls run it with command aleph dev -reload

Thanks that worked!

**Comment by aadamsx** [comment]
> btw, i already released the v0.3.0-alpha.22, some breaking changes included...

@ije you weren't kidding that alpha.22 is breaking: https://github.com/alephjs/aleph.js/issues/222
